lib/offsite_payments/integrations/citrus.rb

```diff
@@ -97,7 +97,7 @@ def invoice_ok?( order_id )
         end
 
         def amount_ok?(order_amount)
-          amount == Money.from_amount(order_amount, currency)
+          gross == order_amount
```

Comment:
I think this would break when `gross` is a string. I guess that's what useful about having a money object, is you can compare without having to make sure casting matches